### DISCUSSION TITLE
You have created a new client application that uses libraries for user authentication or authorization that will soon be deprecated. New clients must use the new libraries instead

### DISCUSSION THREAD

**Discussion by CornellCoetzee** [abacritt/angularx-social-login#533] [Q&A]
Receiving this message in my console when starting up:
You have created a new client application that uses libraries for user authentication or authorization that will soon be deprecated. New clients must use the new libraries instead

Will this package still work?

**Comment by zer0-um** [comment]
In my case, with newly created OAuth2.0 clients this doesn't work anymore

**Reply by CornellCoetzee** [reply]
@gonzalo-lorieto Do you have a workaround or another library for OAuth2.0?
